pkgs/tools/video/wtwitch/default.nix

```diff
@@ -0,0 +1,65 @@
+{ bash
+, coreutils-prefixed
+, curl
+, fetchFromGitHub
+, gnused
+, gnugrep
+, installShellFiles
+, jq
+, lib
+, makeWrapper
+, mplayer
+, mpv
+, ncurses
+, procps
+, scdoc
+, stdenv
+, streamlink
+, sudo
+, vlc
+}:
+stdenv.mkDerivation rec {
+  pname = "wtwitch";
+  version = "2.6.0";
+
+  src = fetchFromGitHub {
+    owner = "krathalan";
+    repo = pname;
+    rev = version;
+    hash = "sha256-KkuXZOquihY3IRVp4FM+AdN3kYi0MqmrXFuNmydTpio=";
+  };
+
+  buildPhase = ''
+    scdoc < src/wtwitch.1.scd > wtwitch.1
+  '';
+
+  nativeBuildInputs = [ scdoc installShellFiles makeWrapper ];
+
+  installPhase = ''
+    installManPage wtwitch.1
+    installShellCompletion --bash --name wtwitch.bash src/wtwitch-completion.bash
+    installShellCompletion --zsh --name _wtwitch src/_wtwitch
+    install -Dm755 src/wtwitch $out/bin/.wrapped/wtwitch
+    makeWrapper $out/bin/.wrapped/wtwitch $out/bin/wtwitch \
+      --set-default LANG en \
+      --prefix PATH : ${lib.makeBinPath ((lib.optionals stdenv.isLinux [ vlc ]) ++ [
```

Comment:
```suggestion
      --prefix PATH : ${lib.makeBinPath (lib.optionals stdenv.isLinux [ vlc ] ++ [
```
didn't test

Reply:
nope, you are on point; the parenthesis is unnecessary